website/static/js/profile.js

```diff
@@ -483,17 +483,34 @@ var SocialViewModel = function(urls, modes) {
     TrackedMixin.call(self);
 
     self.addons = ko.observableArray();
+    
+    self.profileWebsites = ko.observableArray();
+
+    self.profileWebsitesLength = ko.computed(function() {
+        return self.profileWebsites().length;
+    });
+
+    self.hasProfileWebsites = ko.pureComputed(function() {
```

Comment:
This observable seems unnecessary -- could you check if `self.profileWebsites().length > 0` instead?